wdl-lint/tests/lints/inconsistent-newlines/source.errors

```diff
@@ -1,10 +1,10 @@
-note[InconsistentNewlines]: inconsistent newlines detected
+note[InconsistentNewlines]: inconsistent line endings detected
   ┌─ tests/lints/inconsistent-newlines/source.wdl:5:12
   │  
 5 │   version 1.1
   │ ╭───────────^
 6 │ │ 
-  │ ╰^ the first occurrence of a mismatched newline is here
+  │ ╰^ the first occurrence of a mismatched line ending is here
   │  
-  = fix: use either "/n" or "/r/n" consistently in the file
+  = fix: ensure that the same line endings (e.g., `/n` or `/r/n`) are used throughout the file
```

Comment:
Shouldn't these be backslashes?

Reply:
To fix diagnostics with backslashes in paths on Windows, the test baselines normalize all backslashes to slashes, so it's expected the diagnostics are affected this way.

Reply:
Yes! Good catch